assets/src/blocks/Timeline/URLDescriptionHelp.js

```diff
@@ -0,0 +1,22 @@
+const { __ } = wp.i18n;
+
+export const URLDescriptionHelp =
+	__(
+		'Enter the URL of the Google Sheets spreadsheet containing your timeline data.',
+		'planet4-blocks-backend'
+	)
+	+ '<br><a href="https://timeline.knightlab.com/#make" target="_blank" rel="noopener noreferrer">';
```

Comment:
the `;` doesn't allow for the full message to appear :grin:

Reply:
Good catch! 👁️  🔍